Classes/Public/Manager/OCTManagerConstants.h

```diff
@@ -13,31 +13,76 @@ typedef NS_ENUM(NSUInteger, OCTFetchRequestType) {
     OCTFetchRequestTypeMessageAbstract,
 };
 
-typedef NS_ENUM(NSInteger, OCTMessageFileType) {
+typedef NS_ENUM(NSInteger, OCTMessageFileState) {
     /**
      * File is incoming and is waiting confirmation of user to be downloaded.
-     * Please start loading or cancel it with <<placeholder>> method.
+     * Please start loading or cancel it with -OCTSubmanagerFiles saveFile:....
+     * Resumable.
      */
-    OCTMessageFileTypeWaitingConfirmation,
+    OCTMessageFileStateWaitingConfirmation,
 
     /**
      * File is downloading or uploading.
+     * Resumable.
      */
-    OCTMessageFileTypeLoading,
+    OCTMessageFileStateLoading,
 
     /**
-     * Downloading or uploading of file is paused.
+     * Downloading or uploading of file is paused by us.
+     * Resumable.
      */
-    OCTMessageFileTypePaused,
+    OCTMessageFileStatePaused,
 
     /**
      * Downloading or uploading of file was canceled.
+     * Not resumable.
      */
-    OCTMessageFileTypeCanceled,
+    OCTMessageFileStateCanceled,
 
     /**
      * File is fully loaded.
      * In case of incoming file now it can be shown to user.
+     * Not resumable.
      */
-    OCTMessageFileTypeReady,
+    OCTMessageFileStateReady,
+
+    /**
+     * File was interrupted, possibly by the friend going offline,
+     * we crashed, etc.
+     * Pass OCTMessageAbstract to [placeholder] to resume.
+     */
+    OCTMessageFileStateInterrupted
+};
+
+/* Roughly corresponds to TOX_FILE_KIND in toxcore
+ * You should not trust this property as file data is not checked for
+ * valid image/video/whatever data. */
+typedef NS_ENUM(NSInteger, OCTFileUsage) {
+    /**
+     * Standard type of file transfer.
+     */
+    OCTFileUsageData,
+    /**
+     * The file is an image that should be displayed inline, without
+     * the ability for user interaction. (Unimplemented)
+     */
+    OCTFileUsageSticker,
+    /**
+     * Unimplemented.
+     */
+    OCTFileUsageInlinePhoto,
+    /**
+     * Unimplemented.
+     */
+    OCTFileUsageInlineVideo,
+};
+
+/* note: signed type is being used because of Realm. */
+typedef NS_OPTIONS(NSInteger, OCTPauseFlags) {
+    OCTPauseFlagsSelf = 1,
+    OCTPauseFlagsOther = 1 << 1,
```

Comment:
`OCTPauseFlagsOther` sounds like "other" flag, not "other" person. Maybe rename it to `OCTPauseFlagsFriend`?